File: opt-back/src/main/java/com/opt/ssafy/optback/domain/gym/entity/Gym.java
package com.opt.ssafy.optback.domain.gym.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.math.BigDecimal;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@NoArgsConstructor
@Table(name = "gym")
public class Gym {

    @Id
    private int id;

    @Column(name = "phone_number", nullable = false)
    private String phoneNumber;

    @Column(name = "full_address", nullable = false)
    private String fullAddress;

    @Column(name = "road_address", nullable = false)
    private String roadAddress;

    @Column(name = "gym_name", nullable = false)
    private String gymName;

    @Column(name = "latitude", nullable = false)
    private BigDecimal latitude;

    @Column(name = "longitude", nullable = false)
    private BigDecimal longitude;

}

File: opt-fast/models/gym.py
from sqlalchemy import Column, Integer, String, Text, DECIMAL
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

class Gym(Base):
    __tablename__ = "gym"

    id = Column(Integer, primary_key=True, autoincrement=True)
    phone_number = Column(String(20), nullable=True)
    full_address = Column(Text, nullable=True)
    road_address = Column(Text, nullable=True)
    gym_name = Column(String(100), nullable=True)
    latitude = Column(DECIMAL(10, 7), nullable=True)
    longitude = Column(DECIMAL(10, 7), nullable=True)
